src/commands/ext-install.ts

```diff
@@ -57,13 +59,14 @@ export default new Command("ext:install [extensionName]")
   )
   .withForce()
   .option("--params <paramsFile>", "name of params variables file with .env format.")
+  .option("--local", "save to manifest rather than directly install to a Firebase project")
```

Comment:
Agree with manifest not being a great external name. I'll use firebase.json for now but slightly concerned because we are also adding .env files, so it might give a false impression of how to edit the manifest... should we discuss in our weekly meeting?